### ISSUE TITLE
wrong layer type guessed despite correct one being in properties json

### ISSUE THREAD

**Issue by fm3** [scalableminds/webknossos#4673]
we already combine the existing datasource-properties.json with the guesses, right? if the existing one contains a category tag, this one should be used in the suggested displayed json.
An error came up because wk wrongly mistook a 16 bit layer for a segmentation.
Additionally, also the sharing page was blocked by this, tracked in separate issue #4672
